src/BenchmarkDotNet/Loggers/LinqPadLogger.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace BenchmarkDotNet.Loggers
+{
+    public sealed class LinqPadLogger : ILogger
+    {
+        private const string DefaultColor = "";
+
+        // Use Reflection to avoid taking a dependency on the LINQPad assembly
+        static readonly Type linqPadUtil = Type.GetType("LINQPad.Util, LINQPad, Version=1.0.0.0, Culture=neutral, PublicKeyToken=21353812cd2a2db5", throwOnError: true);
+        private static readonly MethodInfo withStyle = linqPadUtil.GetMethod("WithStyle", BindingFlags.Static | BindingFlags.Public);
+
+        public static bool IsAvailable => AppDomain.CurrentDomain.FriendlyName.StartsWith("LINQPad Query Server", StringComparison.OrdinalIgnoreCase);

```

Comment:
Could you please make it lazy?
In BenchmarkDotNet, we are trying to evaluate all the heavy calls only once (on the first usage). It helps to avoid undesired allocations and pauses at random moments.